app/assets/javascripts/Rickshaw.Graph.DragZoom.js

```diff
@@ -0,0 +1,131 @@
+Rickshaw.namespace('Rickshaw.Graph.DragZoom');
```

Comment:
once this is all code reviewed and we're happy with how it works i'm planning on submitting it to the rickshaw repo.